### PR TITLE
Low-e storms

### PR DESCRIPTION
**Pull request by bpark1327**
Addresses https://github.com/NREL/OpenStudio-HEScore/issues/234#issuecomment-966475386

## Pull Request Description

Add a new JSON element, storm_window_type

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] Code changes (must work)
- [ ] Test exercising your feature or bug fix. Check the coverage report in the build artifacts.
- [ ] All other unit tests passing
- [ ] Update translation docs

### CODE REVIEW

File: hescorehpxml/schemas/hescore_json.schema.json [hunk 1]
```diff
@@ -365,6 +365,11 @@
                                                 "$id": "#/properties/building/properties/zone/properties/zone_roof/items/properties/zone_skylight/properties/solar_screen",
                                                 "type": "boolean",
                                                 "description": "Does this window have a solar screen?"
+                                            },
+                                            "storm_window_type": {
+                                                "type": "string",
+                                                "enum": ["clear", "low-e"],
+                                                "description": "Type of storm window"
```
Comment (shorowit) [inline hescorehpxml/schemas/hescore_json.schema.json:372]: It's a little weird to have window in the name but it applies to a skylight. How about:

```suggestion
                                            "storm_type": {
                                                "type": "string",
                                                "enum": ["clear", "low-e"],
                                                "description": "Type of storm for this skylight"
```

File: hescorehpxml/schemas/hescore_json.schema.json [hunk 2]
```diff
@@ -690,6 +695,11 @@
                                             "solar_screen": {
                                                 "type": "boolean",
                                                 "description": "Does this window have a solar screen?"
+                                            },
+                                            "storm_window_type": {
+                                                "type": "string",
+                                                "enum": ["clear", "low-e"],
+                                                "description": "Type of storm window"
```
Comment (shorowit) [inline hescorehpxml/schemas/hescore_json.schema.json:702]: And:

```suggestion
                                            "storm_type": {
                                                "type": "string",
                                                "enum": ["clear", "low-e"],
                                                "description": "Type of storm for this window"
```

File: hescorehpxml/schemas/hescore_json.schema.json [hunk 3]
```diff
@@ -365,6 +365,11 @@
                                                 "$id": "#/properties/building/properties/zone/properties/zone_roof/items/properties/zone_skylight/properties/solar_screen",
                                                 "type": "boolean",
                                                 "description": "Does this window have a solar screen?"
```
Comment (shorowit) [inline hescorehpxml/schemas/hescore_json.schema.json:367]:
```suggestion
                                                "description": "Does this skylight have a solar screen?"
```

**Review by shorowit — COMMENTED** [review]
The implementation looks good, just some minor suggestions.

File: hescorehpxml/schemas/hescore_json.schema.json (hunk 3 above)
Comment (bpark1327) [reply hescorehpxml/schemas/hescore_json.schema.json:367]: Oops 🤦‍♂️. Thanks for catching.

**Review by shorowit — APPROVED** [review]

**Comment by nmerket** [comment]
Waiting on https://github.com/NREL/OpenStudio-HEScore/pull/319

**Pull request merged**